### PR TITLE
PMCP-1735: Update c-title prototype inside the vlanding-video-…

### PR DESCRIPTION
**Pull request by yahilmadakiya**
…showcase

### CODE REVIEW

File: packages/larva-patterns/components/c-tagline/c-tagline.twig [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% if c_tagline_text %}
-	<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}">{{ c_tagline_text }}</p>
+	<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}" {{ c_tagline_data_attr }}>{{ c_tagline_text }}</p>
```
Comment (laras126) [inline packages/larva-patterns/components/c-tagline/c-tagline.twig]: We cannot use this syntax for data attributes due to PHP escaping - 
`<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}" {{ c_tagline_data_attr }}>`

It has to be the value of the data attribute. We should avoid having data attributes on objects and components, and handle JS with `.js-*` class selectors.

File: packages/larva-patterns/modules/vlanding-video-card/vlanding-video-card.prototype.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const c_span = clonedeep( require( './../../components/c-span/c-span.prototype'
 
 module.exports = {
 	vlanding_video_card_permalink_url: "#",
+	vlanding_video_card_link_classes: "lrv-a-unstyle-link",
```
Comment (laras126) [inline packages/larva-patterns/modules/vlanding-video-card/vlanding-video-card.prototype.js]: Single quotes

File: packages/larva-patterns/modules/vlanding-video-showcase/vlanding-video-showcase.prototype.js [hunk 3]
```diff
@@ -2,24 +2,34 @@ const clonedeep = require( 'lodash.clonedeep' );
...
+
+vlanding_video_showcase_card.c_span.c_span_classes += ' lrv-u-display-block lrv-u-padding-b-00 lrv-u-padding-t-050 lrv-u-font-size-12 lrv-u-font-weight-normal';
+
 c_title.c_title_classes = 'lrv-u-font-size-24 lrv-u-font-size-32@desktop lrv-u-padding-b-1';
 c_title.c_title_link_classes = 'lrv-a-unstyle-link lrv-u-color-black';
 c_title.c_title_text = 'Bon Iger: Roseanne is Cancelled and I\'m Like Crying so Hard';
 
-c_tagline.c_tagline_classes = 'lrv-u-color-grey-dark lrv-u-margin-t-00 lrv-a-hidden@mobile-max';
+c_tagline.c_tagline_classes = 'lrv-u-font-size-18 lrv-u-margin-t-00 lrv-a-hidden@mobile-max';
+c_tagline.c_tagline_data_attr = 'data-video-showcase-player-dek';
```
Comment (laras126) [inline packages/larva-patterns/modules/vlanding-video-showcase/vlanding-video-showcase.prototype.js]: I think this is taken care of in the VideoShowcase PR - update this PR to only be adding the span for sponsored badge and the styling

**Comment by yahilmadakiya** [comment]
@laras126 All feedbacks are resolved, Please have a look.

**Pull request merged**